### PR TITLE
Added wrapper for GATK GenomicsDBImport

### PR DESCRIPTION
**Pull request by fgvieira**
Added wrapper for GATK `GenomicsDBImport` and updated the wrapper `GenotypeGVCFs` for compatibility.

### CODE REVIEW

**Comment by jafors** [comment]
@brcopeland has promted a PR (#309) for the same tool, and both have some features that seem important/nicely done to me. Maybe you could join forces and create the "perfect" wrapper. :)

**Comment by brcopeland** [comment]
I looked through both of these and I think it makes sense to use this version; it has the added functionality to update an existing GenomicsDB data store, which I did not implement.

However, there is a change to the `bio/delly` wrapper here which I assume is a mistake.

File: bio/gatk/genotypegvcfs/wrapper.py [hunk 1]
```diff
@@ -14,9 +14,15 @@
 java_opts = get_java_opts(snakemake)
 
 log = snakemake.log_fmt_shell(stdout=True, stderr=True)
+
+input_gvcf = snakemake.input.gvcf
+if os.path.isdir(input_gvcf):
+    input_gvcf = "gendb://" + input_gvcf
+
+
```
Comment (brcopeland) [inline bio/gatk/genotypegvcfs/wrapper.py:22]: The purpose of the PR is to add a wrapper to run GenomicsDBImport.  I think this would appropriately be part of a different PR, which I have done in #310.

File: bio/delly/wrapper.py [hunk 2]
```diff
@@ -8,7 +8,9 @@
 
 
 exclude = (
-    "-x {}".format(snakemake.input.exclude) if snakemake.input.get("exclude", "") else ""
+    "-x {}".format(snakemake.input.exclude)
+    if snakemake.input.get("exclude", "")
+    else ""
```
Comment (brcopeland) [inline bio/delly/wrapper.py:13]: This change does not appear to be relevant to the PR.

File: bio/gatk/genotypegvcfs/environment.yaml [hunk 3]
```diff
@@ -3,5 +3,5 @@ channels:
   - conda-forge
   - defaults
 dependencies:
-  - gatk4 ==4.1.4.1
+  - gatk4 ==4.2.0.0
```
Comment (brcopeland) [inline bio/gatk/genotypegvcfs/environment.yaml:6]: This change appears to be an unnecessary side effect.

**Review by brcopeland — COMMENTED** [review]
Elaboration on prior comments.

File: bio/gatk/genotypegvcfs/wrapper.py (hunk 1 above)
Comment (fgvieira) [reply bio/gatk/genotypegvcfs/wrapper.py:22]: Yes, it could be a different PR, but had to place both on the same branch for testing purposes.

**Review by johanneskoester — APPROVED** [review]
